tests/throughput_test.c

```diff
@@ -74,7 +74,7 @@ throughput_test (int converter, long best_throughput)
 		exit (1) ;
 		} ;
 
-	throughput = lrint (floor (total_frames / duration)) ;
+	throughput = (long) floor (total_frames / duration) ;
```

Comment:
You are chaning the code code here.  The oepration of `lrint` is *different* from the casting to `long`.

Reply:
Can you tell me why? Reason for the change is I wanted to avoid the (private) header float_cast.h. `lrint` rounds according to current rounding mode, then converts to long. Rounding is done here by `floor` so only remaining thing is the conversion. I'm aware that `lrint` might be faster, but a) this is a test and b) it is executed rarely (once? once per individual test? Not in a hot loop anyway)

Reply:
Alternative is to require C99 also in CMake: #68